test/widgets/subscription_list_test.dart

```diff
@@ -186,6 +187,49 @@ void main() {
     check(find.byType(UnreadCountBadge).evaluate()).length.equals(0);
   });
 
+  testWidgets('muted unread badge shows with muted unreads', (tester) async {
+    final stream = eg.stream();
+    final unreadMsgs = eg.unreadMsgs(streams: [
+      UnreadStreamSnapshot(streamId: stream.streamId, topic: 'b', unreadMessageIds: [3]),
+    ]);
+    await setupStreamListPage(tester,
+      subscriptions: [eg.subscription(stream, isMuted: true)],
+      userTopics: [eg.userTopicItem(stream, 'b', UserTopicVisibilityPolicy.muted)],
+      unreadMsgs: unreadMsgs);
+    final finder = find.byWidgetPredicate((widget) => widget is MutedUnreadBadge);
```

Comment:
Can simplify with `find.byType`:
```suggestion
    final finder = find.byType(MutedUnreadBadge);
```

Then just inline into the check below.

For examples, see the similar test cases above about UnreadCountBadge.